### DISCUSSION TITLE
Customise Positron Console (or have the option to use an opposite light/dark theme from editor)

### DISCUSSION THREAD

**Discussion by ddanieltan** [posit-dev/positron#5565] [Q&A]
I would like to customize my Positron to have a dark theme for my editor, but a light theme theme for my console. From https://github.com/posit-dev/positron/discussions/5512, I learnt that I can edit settings such as `positronConsole.background` and `positronConsole.foreground` to override defaults which is great.

But I also realise that there aren't any fields for me to customise the syntax highlights of my code. Which as you can see from this screenshot, is hard to read because I have changed to a light theme based background and foreground colour, but my syntax highlights are still configured for a dark theme.

<img width="2036" alt="image" src="https://github.com/user-attachments/assets/ebc31671-d005-4c0c-893e-7acde80391b1">

Is there anyway for me to specify that I want a light theme for my console?

**Comment by juliasilge** [comment]
Ah unfortunately, there isn't currently a way to set the syntax highlighting separately from the main theme. Thanks for this report, and we will keep an eye out for how common of a request this is! 🙌
